README.md

```diff
@@ -9,13 +9,13 @@ MinIO is High Performance Object Storage released under Apache License v2.0. It
 ### Stable
 ```
 docker pull minio/minio
-docker run -p 9000:9000 minio/minio server /data
+docker run -e MINIO_ACCESS_KEY=samplekey -e MINIO_SECRET_KEY=samplekey -p 9000:9000 minio/minio server /data
```

Comment:
There is a note right below, is there a problem with that?

Reply:
by default these keys are empty, and after we are running docker image in the first page of minio we must enter these keys, and if we do not set this keys, we cant enter keys and can't use the API's

Reply:
It is indicated right after this line as a NOTE: on what needs to be done, I don't think it's appropriate to add this documentation here @RezaOptic